### DISCUSSION TITLE
I cannot reverify education pack

### DISCUSSION THREAD

**Discussion by dmmhlchk** [community/community#65458] [GitHub Education] [answered]
### Hi everyone,

"You were last verified as a student on Aug 25, 2022. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits." I get this message, but now it's August 30th and I can't figure out what's wrong

**Comment by arafajs** [comment, marked as answer]
it usually takes 72 hours for your GitHub benefits to be activated. However, during busy periods like the start of a new semester, it may take a bit longer.

**Reply by yueh0607** [reply]
Sorry, Github has clearly sent an email stating that there is a problem with my application and I need to reapply. However, the webpage display has prevented me from resubmitting my application
Hi, XXXX! You were last verified as a student on Aug 30, 2023 It is not necessary for you to verify at this time There may be a wait period between verification and access to academic benefits

![image](https://github.com/community/community/assets/102401735/e1a73ced-6fc1-4066-8889-d95534aee6ab)
![image](https://github.com/community/community/assets/102401735/ed1fe7bc-6da8-4e48-8792-1a19bd411bc6)

**Comment by rjsvcs** [comment]
I am having the same problem. I'm receiving emails indicating that I need to reverify, but when I try there is an error message stating that there is no need for me to reverify at this time. 

There are indications all over the GitHub Education site that I need to reverify, but, again, when I try it doesn't work.

![image](https://github.com/community/community/assets/22302214/5b977ca6-64e1-4bb0-ada4-70dcf074d975)
![image](https://github.com/community/community/assets/22302214/5fafe61b-a0e3-40fa-b32b-323747e5fcc6)

**Reply by yangsenlin01** [reply]
Me too!!!

**Reply by EnesKeremAYDIN** [reply]
Me too.

**Reply by Konstanty** [reply]
Maybe there is some delay in the expiry in the teacher benefits. Here is a similar issue a few months ago

https://github.com/orgs/community/discussions/55164

(Also I am one week into having the same issue as above)
